docs/source/cli/model-training-inference/real-time-inference.rst

```diff
@@ -0,0 +1,420 @@
+
+``version`` (**Required**)
+>>>>>>>>>>>>>>>>>>>>>>>>>>>
+This field is used to identify the version of a specification, helping to avoid compatibility issues of different
+versions. The current version is ``gs-realtime-v0.1``.
+
+``gml_task`` (**Required**)
+>>>>>>>>>>>>>>>>>>>>>>>>>>>
+This field indicates what graph machine learning task this payload is for. Current specification supports two
+options: 
+
+* ``node_classification``
+* ``node_regression``
+
+``graph`` (**Required**)
+>>>>>>>>>>>>>>>>>>>>>>>>>
+
+This ``graph`` field is similar to fields, e.g., ``graph`` field, in :ref:`GSProcessing input specification
+<gsprocessing_input_configuration>`. It contains three sub-fields, i.e., ``nodes``, ``edges``, and ``targets``.
+
+A ``nodes`` field contains a list of ``node`` fileds. A ``node`` includes the raw input data values
+of a node in the subgraph. It has the following required attributes.
+
+* ``node_type``: string, the raw node type name in a graph. It should be same as these ``node_type`` defined in
+  :ref:`gconstruct JSON specification <gconstruction-json>` or the ``type`` values of ``nodes`` defined in 
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``node_id``: the raw node identifier.
+* ``features``: a dictionary, whose key is a feature name, and its value is the value of the feature.
+  feaure names should be same as these ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+An ``edges`` field contains a list of ``edge`` fields. An ``edge`` includes the raw input data values of an
+edge in the subgraph. It has the following required attributes.
+
+* ``edge_type``: list, the raw edge type name in the format of a list with three elements, which indicate
+  source node type, edge type, and destination edge type. It should be same as these ``relation`` fileds defined
+  in :ref:`gconstruct JSON specification <gconstruction-json>` or these ``type`` values of ``source``
+  ``relation``, and ``dest`` fileds defined in :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``src_node_id``: user defined node identifier for the source node.
+* ``dest_node_id``: user defined node identifier for the destination node.
+* ``features``: a dictionary, whose key is a feature name, and its key is value of the feature. 
+  feaure names should be same as these ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+A ``targets`` field contains a list of target ``node`` or ``edge`` fileds depending on the value of ``gml_task``
+These ``node`` or ``edge`` fileds is same as ``node`` and ``edge`` above, but the features field is not
```

Comment:
Do not understand what you are talking about.

Reply:
Explain this in the new page and add an example to explain. The last sentence is want to make sure the target nodes or edges are in the graph's nodes or edges.